pages/learn/manage/actions.md

```diff
@@ -122,14 +122,16 @@ You may also enable or disable public URLs for a subset of devices by selecting
 
 ### Transfer Application Ownership
 
-You can transfer application ownership between balenaCloud user accounts. In order to transfer application ownership, you will need to perform a series of steps using both the **source** account and **target** account.
+Applications with all their associated devices, releases and members can be transferred to any other balenaCloud [organization][organization]. Application transfers are between a **source** and a **target** organization.
 
-1. Take note of the Application name and balenaCloud username (*in the top-right drop-down*) in the **source** account
-2. In the **target** balenaCloud account, create a new balenaCloud Application using the same Application name
-3. In the **Members** of the Application you just created, add the **source** balenaCloud username as a **Developer**
-4. In the **source** balenaCloud account, select **<Application>** --> **Actions** --> **Transfer This Application**
+Only organization [administrators][administrator] can initiate and complete application transfers. You must coordinate with one of the receiving organization's administrators to perform the following actions:
 
-__Note:__ If the **Transfer This Application** button is grayed out, ensure that you have created an empty Application in the **target** balenaCloud account with the same name as the source Application, and that the **source** username has been added as a **Developer** to the **target** Application.
+1. Take note of the application name in the **source** organization and your balenaCloud username (*in the top-right drop-down*).
+2. Ask an administrator of the **target** balenaCloud organization to create a new application using the same application name.
```

Comment:
NIT: Even though "empty" is also mentioned in the notes, imo it's just a single word and I see no reason to not add it in here as well. Not sure whether this was discussed before though.
```suggestion
2. Ask an administrator of the **target** balenaCloud organization to create a new empty application using the same application name.
```